frontend/src/metabase/nav/components/AppBar/AppBarToggle.styled.tsx

```diff
@@ -1,13 +1,36 @@
 import styled from "@emotion/styled";
+import { css } from "@emotion/react";
 import { color } from "metabase/lib/colors";
 import Icon from "metabase/components/Icon";
 
-export const SidebarButton = styled.button`
+interface SidebarButtonProps {
+  isLogoVisible: boolean;
+}
+
+export const SidebarButton = styled.button<SidebarButtonProps>`
   cursor: pointer;
   display: block;
+
+  && {
+    opacity: ${props => !props.isLogoVisible && 1};
+  }
 `;
 
-export const SidebarIcon = styled(Icon)`
+interface SidebarIconProps {
+  isLogoVisible: boolean;
+}
+
+export const SidebarIcon = styled(Icon)<SidebarIconProps>`
   color: ${color("brand")};
   display: block;
+
+  ${props =>
+    !props.isLogoVisible &&
+    css`
+      color: ${color("text-medium")};
```

Comment:
Previously it wasn't possible to see the toggle button without hovering on desktop, so we didn't have the non-hovered state. But this could happen after we disabled the logo.
https://metaboat.slack.com/archives/C045W5MF84U/p1671814108759669?thread_ts=1671734932.320549&cid=C045W5MF84U